src/_pytest/config/__init__.py

```diff
@@ -1294,6 +1296,18 @@ def _mark_plugins_for_rewrite(self, hook) -> None:
         for name in _iter_rewritable_modules(package_files):
             hook.mark_rewrite(name)
 
+    def _configure_python_path(self):
+        # `pythonpath = a b` will set `sys.path` to `[a, b, x, y, z, ...]`
+        for path in reversed(self.getini("pythonpath")):
+            sys.path.insert(0, str(path))
+        self.add_cleanup(self._unconfigure_python_path)
+
+    def _unconfigure_python_path(self):
```

Comment:
This is not being called?

Also:

```suggestion
    def _unconfigure_python_path(self) -> None:
```